plugins.d/python.d.plugin

```diff
@@ -515,6 +519,9 @@ def parse_cmdline(directory, *commands):
             mods.append(cmd.replace(".chart.py", ""))
         else:
             try:
+                # FIXME for some reason this overwrites the module configuration
```

Comment:
I have probably misunderstood you. And I have a better solution than previous one:
replace
```py
else:
    try:
        BASE_CONFIG['update_every'] = int(cmd)
        OVERRIDE_UPDATE_EVERY = True
        debug(PROGRAM, "overriding update interval to", str(int(cmd)))
    except ValueError:
        pass

```
```py
for cmd in commands[1:]:
    try:
        BASE_CONFIG['update_every'] = int(cmd)
        if DEBUG_FLAG:
            OVERRIDE_UPDATE_EVERY = True
            debug(PROGRAM, "overriding update interval to", str(int(cmd)))
    except ValueError:
        pass
```